plugins/ti-dmc/README.md

```diff
@@ -0,0 +1,39 @@
+# Texas Instruments TPS6598x
+
+## Introduction
+
+The TPS65987D and TPS65988 are Type-C Port Switch / Power Delivery (PD) Controller devices.
```

Comment:
should the plugin be called `ti-tps6598x` instead?

Reply:
Agreed, renamed.